packages/tangle/tangle.go

```diff
@@ -199,6 +263,9 @@ type Events struct {
 
 	// Error is triggered when the Tangle faces an error from which it can not recover.
 	Error *events.Event
+
+	// Info is triggered when the tangle should log the info provided.
+	Info *events.Event
```

Comment:
This is never triggered, so why was this added? It is just a left-over debug artifact or is it a valuable addition?

Reply:
I think it's just for debugging. Let's remove it.